rf-notebook/README.md

```diff
@@ -0,0 +1,32 @@
+# RasterFrames Jupyter Notebook Docker Image
+
+RasterFrames provides a Docker image with a Jupyter Notebook pre-configured with RasterFrames in the PySpark kernel.
+
+## Launching RasterFrames Notebook from Source
+
+First, have Docker installed.
+
+Then, from the source root directory, run
```

Comment:
maybe here say 

```suggestion
Then, from the repo root directory, run
```

Source root could be interpreted as `rf-notebooks/src`?

Reply:
@vpipkt It needs to be the `rasteframes` directory.... what do you think the right name is?